docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md

```diff
@@ -59,6 +59,31 @@ For `dataframe` Data Assets, the `dataframe` is always specified as the argument
 ```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py build_batch_request_with_dataframe"
 ```
 
+## Extract a Batch from a Batch Request (Optional)
+
+You can use the Python slice function to remove a subset of data from a Batch Request and use a specific selection of records to build Metrics, Validations, and Profiles.
+
+1. Run the following code to retrieve an entire table of data from a SQL datasource:
+
+    ```python
+    table_asset = datasource.add_table_asset(name=asset_name, table_name=asset_table_name)
+    ```
+2. Run the following code to define the column to slice:
```

Comment:
So there are other ways to slice the data, in addition to column. You can do it by time, year, etc. I don't know if we want to mention that somewhere, or add in the language that slicing the data by column is one example in the direction here
![Screenshot 2023-08-31 at 7 51 51 AM](https://github.com/great-expectations/great_expectations/assets/133805639/de60a480-9b36-4ad5-9488-8bd8e93d5162)